2018-12-26/KBitsCombination.java

```diff
@@ -15,21 +15,33 @@
 */
 class KBitsCombination {
   public static void main(String[] args) {
+    ArrayList<int[]> combinations;
+
     System.out.println(" ================================= ");
     System.out.println(" Case for n = 8 and k = 3 ");
-    KBitsCombination.findAllCombinations(8, 3);
+    combinations = KBitsCombination.findAllCombinations(8, 3);
+    
+    for (int[] c : combinations) {
+      System.out.println(java.util.Arrays.toString(c));
+    }
+
     System.out.println(" ================================= ");
     System.out.println("  Case for n = 4 and k = 2 ");
-    KBitsCombination.findAllCombinations(4, 2);
+    combinations = KBitsCombination.findAllCombinations(4, 2);
+
+    for (int[] c : combinations) {
```

Comment:
`for (int[] c : findAllCombinations(4, 2)) {` should work. You don't need the `KBitsCombination.` prefix because it's referring to the current class.